src/test/java/redis/clients/jedis/JedisClusterTest.java

```diff
@@ -216,14 +216,14 @@ public void testMigrate() {
         node2.set("e", "e");
       } catch (JedisMovedDataException jme) {
         assertEquals(15363, jme.getSlot());
-        assertEquals(new HostAndPort(LOCAL_IP, nodeInfo3.getPort()), jme.getTargetNode());
+        assertEquals(new HostAndPort(JedisClusterTestUtil.getClusterIp(3), 6379), jme.getTargetNode());
```

Comment:
Same general comment re ports - throughout.

Reply:
Yes, we have to do something about this. It's so spaghetti right now, implied relations between ports and stuff. My intention was to leave that for another PR, go step by step. Here I did a minimal mapping to the new Docker IPs and ports.